scripts/api/data/metadatablocks/citation.tsv

```diff
@@ -325,3 +326,9 @@
 	language	Zhuang, Chuang	zha	183	zha	za	Zhuang	Chuang
 	language	Zulu	zul	184	zul	zu
 	language	Not applicable		185											
+	publicationRelationType	IsCitedBy	RT1	1											
```

Comment:
@jggautier - which of the relationships DataCite allows should be here? This is a ~minimal list right now.

Reply:
Hi @qqmyers. We're not sure. I'm planning for the UX WG to research this as part of its second design sprint and possibly in collaboration with the NIH GREI group if they start their work in time.

And the scope of the second design sprint is broader than the changes I see in this PR. They're about improving how the metadata about different types of research objects are related to the deposit.

Could we wait until the UX WG can start the second design sprint? If the first design sprint goes well, it'll end in mid-July, and the second design sprint can start a week or two later.

Reply:
This PR is coming from QDR where we'll make some choice and move forward with deployment, probably just dealing with related Publications to start. What the scope, priority and timescale are for Harvard or the community is not something I can answer beyond echoing your idea before of not slowing things down to get it perfect. Nominally it is easier to add additional items than to delete them (and figure out what to do with entries that used those dropped terms), so if we want this to move forward quickly, I'd suggest going with a non-controversial minimal set. That said, with the 6.3 release, July 4th,  and other things going on, I don't know that this will move forward too quickly anyway so it may be that waiting for the UX WG input is fine.

Reply:
Yeah, I think the conversations the community has had so far have stalled because the community hasn't been able to focus on gaining a shared understanding about why and how we want to describe the relationships between research objects. And I'm hoping that the UX WG and the design sprints we're trying are a way for the community, and for me to be honest, to commit the time to focus on this and other challenges when people use these metadata fields.

Reply:
Hopefully this PR will help advance the community's thinking on these relationships!